src/lambdapi.ml

```diff
@@ -111,6 +109,30 @@ let lsp_server_cmd : config -> bool -> string -> unit =
   let run _ = init cfg; Lsp.Lp_lsp.main standard_lsp lsp_log_file in
   Console.handle_exceptions run
 
+(** Printing a decision tree. *)
+let decision_tree_cmd : config -> string -> unit = fun cfg qsym ->
+  let run _ =
+    init cfg;
+    let (mp, sym) =
+      let rec get red mp =
+        match red with
+        | [sym] -> (mp, sym)
+        | m::ms -> get ms (m::mp)
+        | []    -> assert false
+      in
+      let (mp, sym) = get (String.split_on_char '.' qsym) [] in
+      (List.rev mp, sym)
+    in
+    Timed.(verbose := 0); (* To avoid printing the "Checked ..." line *)
```

Comment:
Please move this before `init cfg;` so that the user can override. Otherwise the user's input for the `--verbose` option is ignored completely. You just want to modify the default behaviour here.

Reply:
This is intentional, you don't want the "Checked ..." to pipe to `dot`

Reply:
Yeah I know, but in that case it is weird to allow the `--verbose` option here. I would say, apply my comment for now (we might still want to use `--verbose` together with that command, who knows?). I can always come back later and improve this by changing the available options for this command. (I am planning to work on other commands soon, so I will need to do that anyway. No need to do that now.)

Reply:
Ok. It's a bit weird as the option *must* be used with `--verbose 0` to work properly. Perhaps I can default it to 0 in that command?

Reply:
That is exactly what I was suggesting by moving the update.

Reply:
Oh but `--verbose` has the default option... So actually you need to make it so that verbose only carries on option.